apps/vanilla/src/pages/index.html

```diff
@@ -9,7 +9,7 @@
   <body class="default-layout">
     <h1 class="title">Main page ⚡</h1>
     <div class="links">
-      <a href="example/">Go to example page</a>
+      <a href="blackJack/">Go to BlackJack</a>
```

Comment:
I'm not sure whether it was written in the task or not but usually we have cabab case in link:
```suggestion
      <a href="black-jack/">Go to BlackJack</a>
```